exonum/src/runtime/mod.rs

```diff
@@ -341,13 +341,17 @@ pub struct ArtifactProtobufSpec {
     ///
     /// The common interface entry point is always in the `service.proto` file.
     pub sources: Vec<ProtoSourceFile>,
+    /// List of service's proto include files.
```

Comment:
What is the difference between include files and sources? E.g. I have `service.proto`, `my-awesome-helpers.proto` and `my-data-types.proto`. `my-awesome-helpers.proto` is used both in `service.proto` and `my-data-types.proto`, while `service.proto` also includes `my-data-types.proto`. What are includes and what are sources here?
How the user should communicate this to the runtime?

Reply:
nit: documentation of `sources` field is invalid, as it mentions `tuple`

Reply:
The difference is important when you compile proto files. To complie service.proto you need to
provide `service.proto` itself and two include files `awesome-helpers.proto` and `my-data-types.proto`.
> How the user should communicate this to the runtime?
`ProtobufGenerate` will create appropriate constants in service's generated proto_mod.rs